ini_multiline_value_fix_test.go

```diff
@@ -0,0 +1,60 @@
+package ini_test
```

Comment:
I think `ini_python_multiline_test.go` is a better name.

Reply:
Sure, no problem